services/comm/app/models/template.rb

```diff
@@ -11,6 +11,8 @@ def initialize_properties
 
   # See: https://www.stuartellis.name/articles/erb/
   def render(user, campaign)
+    return content if user.nil? || campaign.nil?
```

Comment:
This is not necessarily true. What if we don't want to refer to the campaign in the sms? Or user?
I think that, at most, one could argue that this is useless if both are nil